grafanalib/core.py

```diff
@@ -1238,6 +1240,7 @@ def to_json_data(self):
             'stack': self.stack,
             'steppedLine': self.steppedLine,
             'tooltip': self.tooltip,
+            'thresholds': self.thresholds,
```

Comment:
I believe that `thresholds` is configurable only if there is no alert set, otherwise `thresholds` will be simply ignored.

Reply:
I have added a printed waning and something to the docs

Reply:
It should produce a valid config for the dashboard, `thresholds` will be just ignored. So I think a warning is enough.

Reply:
Also, we can define it in `else` of `if self.alert` block simply ignoring when we have an alert defined.